### DISCUSSION TITLE
Development machine

### DISCUSSION THREAD

**Discussion by Jwerfel** [coder/code-server#2289] [General]
I'm curious what people are using for their development system when using code-server. I am used to running a high power development environment but by moving that to the server I should be able to use a more basic system. It seems like a Chromebook is the most logical device to use but not all Chromebooks are created equal so I'm curious what others are using and what the minimum system specs would be.

Thanks

**Comment by bpmct** [comment]
I use an ASUS-A201 Chromebook and see no performance issues. From my experience with various Dell and Asus chromebooks, they still work well with all advanced web-apps such including code-server.
